src/doc_scan_service/doc.scan.constants.js

```diff
@@ -13,6 +13,7 @@ module.exports = Object.freeze({
   ID_DOCUMENT_FACE_MATCH: 'ID_DOCUMENT_FACE_MATCH',
   ID_DOCUMENT_COMPARISON: 'ID_DOCUMENT_COMPARISON',
   THIRD_PARTY_IDENTITY: 'THIRD_PARTY_IDENTITY',
+  THIRD_PARTY_IDENTITY_FRAUD_1: 'THIRD_PARTY_IDENTITY_FRAUD_1',
```

Comment:
I think the value of this constant should be `CIFAS_IDENTITY`.

This corresponds to the `type` of a `check`, as defined in the linked ticket, belonging to the Doc Scan API team: https://lampkicking.atlassian.net/browse/DOCS-1554.

PS: THIRD_PARTY_IDENTITY_FRAUD`_1` would anyway be a pretty bad name IMO, as it would be very confusing, and not descriptive when compared to the normal `THIRD_PARTY_IDENTITY_FRAUD`, or other potential future THIRD_PARTY_IDENTITY_FRAUD`_n`

Reply:
@manuel-yoti @liamd-yoti This won't be `CIFAS_IDENTITY`.  Those tickets were written before product asked us to change it, and we haven't yet written tickets for how this will be exposed on the public API.

Reply:
I see, think we may have jumped on this one a bit early then. Thanks for clearing that up 👍

Reply:
Thanks for pointing this out @MrBurtyyy 
Can we continue this thread in a Jira ticket ? 

I left a comment about this topic in here: https://lampkicking.atlassian.net/browse/SDK-2118

Reply:
So it has been settled to be: `THIRD_PARTY_IDENTITY_FRAUD_1`, will revert.